### DISCUSSION TITLE
OpenOutpaint running on same port as SD

### DISCUSSION THREAD

**Discussion by unkleb** [zero01101/openOutpaint#238] [Q&A]
I've reinstalled your marvelous software, this time straight from github.  It seems to be working almost perfectly.  However, I can not get the image from OpenOutpaint back to SD img2img.  Nothing seems to work to get the final image back to SD.  The one thing I notice though, is that OpenOutpaint is running on the same port as SD.  I have made the change to my web-user.bat file and followed all the other directions.  When I use the extension to widen an image, it looks perfect!  What a wonderful product!  But I am stumped after that.  I can save or download the original, but just can't do anything with the edited image.

Am I doing something wrong, or just confused?

Thanks.

**Comment by zero01101** [comment]
hey, i'm really sorry for the delay, i totally missed this :| 
so by default, if you're running openOutpaint standalone, the openOutpaint.bat/sh should open a server on port 3456, but if you're running the extension, it doesn't so much run on the same port as it just runs within the web server hosting A1111 webUI 

given your context, i'll presume you're using the extension, and i think you're looking for the [marquee select's](https://github.com/zero01101/openOutpaint/wiki/Manual#marquee-select) send-to functionality - after you select a region using that tool, the dropdown select options in the context menu will activate, allowing you to choose a destination to send the content selected
